### DISCUSSION TITLE
This user account is blocked.

### DISCUSSION THREAD

**Discussion by Timed-AutoMaton** [vercel/community#3613] [Account Issue] [answered]
### Question

Hi,

I have recently tried to log in to Vercel via my GitHub account. But, I'm getting this error "This user account is blocked."

Please help!
![Screenshot_1](https://github.com/vercel/community/assets/97603324/00b8be5e-c754-4ef3-acd2-5c60ca9454ae)
![Screenshot_2](https://github.com/vercel/community/assets/97603324/f556b958-95aa-40f8-bcc8-f81de30c6a10)

**Comment by amyegan** [comment, marked as answer]
Please email [registration@vercel.com](mailto:registration@vercel.com) and include the email and username for the account. Vercel employs multiple layers of security and verification. Providing additional information about your account and the project you are deploying can help accelerate this process.
